lib/std/rand/Romu192.zig

```diff
@@ -0,0 +1,141 @@
+// Website: romu-random.org
+// Reference paper:   http://arxiv.org/abs/2002.11331
+
+// important note: this RNG exposes all of it's state in the random sequences generated, so it is trivial to write a state prediction function.
+// it's use is recommended only in context where there is no risk of prediction, e.g. it is perfectly safe for a monte carlo simulation.
```

Comment:
```suggestion
// Beware: this PRNG is trivially predictable. While fast, it should *never* be used for cryptographic purposes.
```
Perhaps a reword for clarity here?

Reply:
@francescoalemanno Just making sure you didn't miss these suggestions